examples/apps/echo/client/echo_client.py

```diff
@@ -237,10 +239,24 @@ def Main(args=None):
 		json.dumps(res, indent=4)
 	))
 	if "result" in res:
-		decrypted_res = utility.decrypted_response(
-			json.dumps(res), session_key, session_iv)
-		logger.info("\nDecrypted response:\n {}".format(decrypted_res))
+		sig_obj = signature.ClientSignature()
+		status = sig_obj.verify_signature(json.dumps(res), worker_obj)
```

Comment:
Can we change the input parameters of verify_signature() to use json dictionary instead of json string. This will avoid unnecessary conversion from json str to json dictionary and vice-versa. 
Also entire worker_obj need not be passed to the function.

Reply:
worker_obj is passed because verify_signature is checking for hashing algorithm, signing algorithm and verification key from worker_obj thats it is passed as argument.

Reply:
Would it make more sense to pass just the required fields if its not too much of an ask ?